src/validations/prefab.ts

```diff
@@ -26,7 +28,8 @@ const schemaProvider = (
       .valid(...ICONS)
       .required(),
     category: Joi.string().required(),
-    type: Joi.string().valid('page'),
+    type:
+      prefabType === 'partial' ? Joi.forbidden() : Joi.string().valid('page'),
```

Comment:
The new types from my previous comment will prevent this kind of stuff as well 😛